odoo/addons/base/rng/calendar_view.rng

```diff
@@ -26,6 +26,7 @@
             <rng:optional><rng:attribute name="create"/></rng:optional>
             <rng:optional><rng:attribute name="delete"/></rng:optional>
             <rng:optional><rng:attribute name="scales"/></rng:optional>
+            <rng:optional><rng:attribute name="banner_route"/></rng:optional>
```

Comment:
Commit "[IMP] base: add banner_route to calendar view attributes": could be great to at least highlight why we need this attribute :) commit message explains what we do but not why (which is the interesting part)

Reply:
What is the attribute, why is enabling the banner (already in message).
Why we need a banner on calendar view is explained in the overarching PR message that will be merge commit message, right? 
I'll add it here too, but I won't add why we need an onboarding :D